crates/swc_css_prefixer/tests/fixture/duplicate-at-rule-3/output.css

```diff
@@ -1,3 +1,4 @@
+@-webkit-keyframes anim {}
```

Comment:
Expected, because the second `@-webkit-keyframes` after `@keyframes` (`autoprefixer` does the same), anyway we will remove it in future when we implemented removing outdated prefixes (after resolve remain TODOs)